ocf_datapipes/utils/consts.py

```diff
@@ -336,6 +336,9 @@ class BatchKey(Enum):
     "lcc": 47.90454236572895,
     "mcc": 44.22781694449808,
     "hcc": 32.87577371914454,
+    "dlwrf": 319,
```

Comment:
I would prefer separating out MetOffice NWP and GFS NWP values, each one will have different values as they are calculated differently.

Reply:
thats a good idea